### ISSUE TITLE
Error on https://flashstorage.games/embed/french-chef-real-cooking

### ISSUE THREAD

**Issue by ErzenXz** [ruffle-rs/ruffle#10784]
# Error Info
Error name: Error
Error message: panicked at 'No caller domain available - use Activation::from_domain when constructing your domain', core/src/avm2/activation.rs:676:28
Error stack:
```
Error: panicked at 'No caller domain available - use Activation::from_domain when constructing your domain', core/src/avm2/activation.rs:676:28
    at O.n.wbg.__wbg_new_15d3966e9981a196 (https://flashstorage.games/public/js/emulators/ruffle/core.ruffle.dcbb4a112d0621488d8f.js:1:47949)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hbac0af9b7f44ed2e (wasm://wasm/02a77236:wasm-function[8135]:0x7ee656)
    at std::panicking::rust_panic_with_hook::hc53aea0352e77326 (wasm://wasm/02a77236:wasm-function[5463]:0x7272f5)
    at core::panicking::panic_fmt::hf4a9df75710ece83 (wasm://wasm/02a77236:wasm-function[8017]:0x7e92f3)
    at core::option::expect_failed::h8f07431afd4c5625 (wasm://wasm/02a77236:wasm-function[9308]:0x815ae9)
    at ruffle_core::avm2::function::Executable::exec::h459ce63d33fcd2f3 (wasm://wasm/02a77236:wasm-function[921]:0x2f9159)
    at <ruffle_core::avm2::object::Object as ruffle_core::avm2::object::TObject>::call::h4aec07be6c66dfa0 (wasm://wasm/02a77236:wasm-function[945]:0x3091ec)
    at ruffle_core::avm2::events::dispatch_event_to_target::hc9fd98c358ca4352 (wasm://wasm/02a77236:wasm-function[886]:0x2e3abd)
    at ruffle_core::avm2::events::dispatch_event::h48ad47345cdb0200 (wasm://wasm/02a77236:wasm-function[1040]:0x33e7e5)
    at ruffle_core::avm2::Avm2::dispatch_event::hd8c3385336e5af8e (wasm://wasm/02a77236:wasm-function[1248]:0x3a0ad7)
```

# Player Info
Allows script access: true

# Page Info
Page URL: https://flashstorage.games/embed/french-chef-real-cooking?emulator=ruffle
SWF URL: https://flashstorage.games/public/storage/games/January2022/frenchchefrealcooking.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-04-17
Channel: nightly
Built: 2023-04-17T00:19:12.508Z
Commit: e8aa620767a4eea9cd1a40ceba0a6ca7ffee634e
Is extension: false

# Metadata
width: 720
height: 480
frameRate: 24
numFrames: 70
swfVersion: 10
backgroundColor: undefined
isActionScript3: true
uncompressedLength: 5667824

**Comment by Lord-McSweeney** [comment]
See https://github.com/ruffle-rs/ruffle/issues/10748.

**Comment by Lord-McSweeney** [comment]
Fixed by #10811.
